examples/core/input/0_input_keyboard/main.cpp

```diff
@@ -18,14 +18,18 @@ int main() {
         std::cout << "key press\n";
     };
 
-    auto input = window->get_input();
+    InputHandler input_handler = InputHandler();
```

Comment:
Same as https://github.com/Maksasj/bebone/pull/123#discussion_r1660209425

Reply:
That's my code style. I don't think it harms our code in any way. For me, the `auto` keyword reduces readability. I don't like to use it, except when there is `std::shared_pointer`.